Dmin_autocalculator.py

```diff
@@ -0,0 +1,230 @@
+    # resets tracking directories
+    try:
+        os.mkdir(save_dir)
+    except FileExistsError:
+        pass
+    try:
+        os.mkdir(save_dir + '\\trackAnalysis')
+        os.mkdir(save_dir + '\\trackAnalysis\\backgrounds')
+        os.mkdir(save_dir + '\\trackAnalysis\\cells')
+    except FileExistsError:
+        shutil.rmtree(save_dir + '\\trackAnalysis')
+        os.mkdir(save_dir + '\\trackAnalysis')
+        os.mkdir(save_dir + '\\trackAnalysis\\backgrounds')
+        os.mkdir(save_dir + '\\trackAnalysis\\cells')
+
+    # runs analysis for each given directory
+
+    for dir in directories:
+        print("Analysing " + str(directories.index(dir) + 1) + "/" + str(len(directories)) + "  " + dir)
+        notebook_analysis = trackAnalysis.analysisNotebook(software, mask_words, dir,
+                                                           dir + r'\cells\rois\cell_sizes.LOG',
+                                                           pixel_size, background_size,
+                                                           camera_integration_time, n_points, MSD_f_area, dof, min_dD,
+                                                           min_tra_len,
+                                                           id_type)
+        notebook_analysis.run_analysis()
+        notebook_analysis.save_analysis()
+        for file in get_matching_files(dir, '.h5', 'background'):
+            try:
+                shutil.move(file, save_dir + '\\trackAnalysis\\cells')
+            except shutil.Error:
+                print(
+                    'File ' + file + ' already exists in the trackAnalysis\\cells directory. Please check for duplicate files. Skipping...')
+                pass
+    # runs analysis for given backgrounds
+    for bg in background:
+        print("Analysing " + str(background.index(bg) + 1) + "/" + str(len(background)) + "  " + bg)
+        notebook_analysis = trackAnalysis.analysisNotebook(software, mask_words, bg, '', pixel_size, background_size,
+                                                           camera_integration_time, n_points, MSD_f_area, dof, min_dD,
+                                                           min_tra_len,
+                                                           id_type)
+        notebook_analysis.run_analysis()
+        notebook_analysis.save_analysis()
+        for file in get_matching_files(bg, '.h5', 'cell'):
+            try:
+                shutil.move(file, save_dir + '\\trackAnalysis\\backgrounds')
+            except shutil.Error:
+                print(
+                    'File ' + file + ' already exists in the trackAnalysis\\background directory. Please check for duplicate files. Skipping...')
+                pass
+    # creates the statistics notebook depending on whether backgrounds are given or not
+    if len(background) == 0:
+        notebook_statistics = dlp.statisticsNotebook(save_dir + '\\trackAnalysis\\cells', '',
```

Comment:
could you either use '' or "" throughout? Preferably "" to match the other code in SPTAnalyser